subject_files_status_for_dpdash.py

```diff
@@ -45,19 +43,36 @@ def get_summary_from_phoenix(phoenix_dir: Path) -> pd.DataFrame:
     df['level0'] = df.p.apply(lambda x: x.parent.parent.parent.name)
     df['level1'] = df.p.apply(lambda x: x.parent.name)
 
-    df['pronet_surveys'] = df.p.apply(lambda x: len(list((x / 'surveys').glob('*Pronet.json'))) > 0)
-    df['upenn_surveys'] = df.p.apply(lambda x: len(list((x / 'surveys').glob('*UPENN.json'))) > 0)
-    df['eeg'] = df.p.apply(lambda x: len(list((x / 'eeg').glob('*zip'))) > 0)
-    df['eeg_ss'] = df.p.apply(lambda x: (x / 'eeg' / f'{x.name}.Pronet.Run_sheet_eeg.csv').is_file())
-    df['actigraphy'] = df.p.apply(lambda x: len(list((x / 'actigraphy').glob('*zip'))) > 0)
-    df['actigraphy_ss'] = df.p.apply(lambda x: (x / 'actigraphy' / f'{x.name}.Pronet.Run_sheet_actigraphy.csv').is_file())
-    df['mri'] = df.p.apply(lambda x: len([x for x in (x / 'mri').glob('*') if x.is_dir()]) > 0)
-    df['mri_ss'] = df.p.apply(lambda x: (x / 'mri' / f'{x.name}.Pronet.Run_sheet_mri.csv').is_file())
-    df['interviews'] = df.p.apply(lambda x: len([x for x in (x / 'interviews').glob('*') if x.is_dir()]) > 0)
-    df['interviews_ss'] = df.p.apply(lambda x: (x / 'interviews' / f'{x.name}.Pronet.Run_sheet_interviews.csv').is_file())
-    df['mind_phone'] = df.p.apply(lambda x: len( [x for x in (x / 'phone').glob('*_activity_*json')] ))
-    df['mind_audio'] = df.p.apply(lambda x: len( [x for x in (x / 'phone').glob('*_activity_*_sound_*.mp3')] ))
-    df['mind_sensor'] = df.p.apply(lambda x: len( [x for x in (x / 'phone').glob('*_sensor_*json')] ))
+    # surveys
+    df['surveys'] = df.p.apply(
+        lambda x: _is_file(x, 'surveys', '*Pronet.json') or 
+                  _is_file(x, 'surveys', '*.csv'))
```

Comment:
#### Note for Tashrif for DPdash

If `df['surveys']` is tracked like this, in the `ProNET` configuration card, we have to label it `UPENN Forms Raw Data` and in the `PRESCIENT` configuration card, we have to call it `RPMS Forms Raw Data`. But on the bright side, `*json` and `*csv` files wouldn't occur for the same subject.